src/LinAlg/SystemMatrix.h

```diff
@@ -279,6 +279,9 @@ class SystemMatrix
   //! \brief Truncates all small matrix elements to zero.
   virtual bool truncate(Real = Real(1.0e-16)) { return false; }
 
+  //! \brief Multiplication with a scalar.
+  virtual void mult(Real) {}
```

Comment:
pure virtual?

Reply:
OK, then have to implement for SPRMatrix also.